releasenotes/notes/rename-kubelet-corecheck-8f28eff881fde78a.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Rename ``kubelet_core`` check to ``kubelet`` and change metrics prefix 
+    from ``kubernetes_core`` to ``kubernetes`` to allow as replacement for 
+    Python ``kubelet`` check.
```

Comment:
```suggestion
    Rename the ``kubelet_core`` check to ``kubelet`` and change the metrics prefix 
    from ``kubernetes_core`` to ``kubernetes`` so that it can replace the
    Python ``kubelet`` check.
```